cmd/version.go

```diff
@@ -1,51 +1,46 @@
 package cmd
 
 import (
-	"fmt"
-	"runtime/debug"
+	"context"
+	"time"
 
 	"github.com/spf13/cobra"
+	"github.com/mcpjungle/mcpjungle/internal/version"
 )
 
-const defaultVersion = "dev"
+// getServerVersion attempts to fetch the server version from the configured server.
+// Returns the version string and a boolean indicating success.
+func getServerVersion() (string, bool) {
+	// Try to get server metadata with timeout
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
 
-// Version can be overridden at build time using:
-// go build -ldflags="-X 'github.com/mcpjungle/mcpjungle/cmd.Version=v1.2.3'"
-var Version = defaultVersion
-
-// getVersion returns the CLI version string.
-func getVersion() string {
-	if Version != "" && Version != defaultVersion {
-		return normalizeVersion(Version)
-	}
-
-	if info, ok := debug.ReadBuildInfo(); ok && info.Main.Version != "" && info.Main.Version != "(devel)" {
-		return normalizeVersion(info.Main.Version)
+	metadata, err := apiClient.GetServerMetadata(ctx)
+	if err != nil {
+		return "", false
 	}
 
-	return defaultVersion
-}
-
-// normalizeVersion ensures a consistent version format:
-// - If version starts with a digit (e.g., "1.2.3"), prefix with 'v' → "v1.2.3"
-// - Leave values starting with 'v' or non-semver strings untouched
-func normalizeVersion(v string) string {
-	if v == "" {
-		return v
-	}
-	if v[0] >= '0' && v[0] <= '9' {
-		return "v" + v
-	}
-	return v
+	return metadata.Version, true
 }
 
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print version information",
 	Run: func(cmd *cobra.Command, args []string) {
 		// We want the extra newline for proper formatting
-		fmt.Print(asciiArt) //nolint:staticcheck
-		fmt.Printf("MCPJungle %s\n", getVersion())
+		cmd.Print(asciiArt) //nolint:staticcheck
+		
+		// Display CLI version
+		cliVersion := version.GetVersion()
+		cmd.Printf("CLI Version: %s\n", cliVersion)
+		
+		// Try to fetch server version
+		serverVersion, ok := getServerVersion()
+		if ok {
+			cmd.Printf("Server Version: %s\n", serverVersion)
+		} else {
+			cmd.Printf("Server Version: Couldn't retrieve Server version at this time.\n")
```

Comment:
One more thing. Also add at the end: `(server url: <server url>)`.
Basically this tells the user what url the cli tried to use to reach the server.
You can easily get the url using `apiClient.BaseURL()` method.